src/DynamoPackages/PackageDirectoryBuilder.cs

```diff
@@ -212,19 +213,12 @@ private void WritePackageHeader(Package package, IDirectoryInfo rootDir)
             fileSystem.WriteAllText(headerPath, pkgHeaderStr);
         }
 
-        internal void CopyFilesIntoRetainedPackageDirectory(IEnumerable<IEnumerable<string>> contentFiles, IEnumerable<string> markdownFiles, IDirectoryInfo rootDir, out List<string> dyfFiles)
+        internal void CopyFilesIntoRetainedPackageDirectory(IEnumerable<IEnumerable<string>> contentFiles, IEnumerable<string> markdownFiles, string oldDir, IDirectoryInfo rootDir, out List<string> dyfFiles)
```

Comment:
oldDir is not super clear to me as a name ... maybe sourcePackageDir?

Reply:
Done